### DISCUSSION TITLE
How to add graph to slide?

### DISCUSSION THREAD

**Discussion by AkagiYui** [vuepress-theme-hope/vuepress-theme-hope#3041] [Q&A] [answered]
Can I add some graph to slide?
https://theme-hope.vuejs.press/guide/markdown/chart.html
https://theme-hope.vuejs.press/guide/markdown/echarts.html
https://theme-hope.vuejs.press/guide/markdown/mermaid.html
https://theme-hope.vuejs.press/guide/markdown/flowchart.html

**Comment by Mister-Hope** [comment, marked as answer]
No, reveal use it's own parser to parse markdown content at runtime.
